src/test/resources/com/puppycrawl/tools/checkstyle/astprinter/InputNoUnnecessaryTextInJavadocAst.javadoc

```diff
@@ -1,4 +1,4 @@
 * <p>
 * My {@code Class}
 * </p>
-* @see something
\ No newline at end of file
+* @see something
```

Comment:
adding "\n" is not required, please revert it as it cause useless changes in expected AST